backend/handler/question_groups.go

```diff
@@ -28,6 +28,30 @@ func (s *Server) GetQuestionGroups(e echo.Context) error {
 	return e.JSON(http.StatusOK, res)
 }
 
+func (s *Server) GetQuestionGroup(e echo.Context, questionGroupID int, params GetQuestionGroupParams) error {
```

Comment:
`int`型ではなく`QuestionGroupId`型を使ってほしいです :pray:
（api.gen.goを見ると`QuestionGroupId = int`として定義されているので同じ型ではあるんですが、自動生成された型を使っておくと万が一将来`int`型でなくなってもここを書き換える必要がないというメリットがあります）
```suggestion
func (s *Server) GetQuestionGroup(e echo.Context, questionGroupID QuestionGroupId, params GetQuestionGroupParams) error {
```

Reply:
了解です！